crates/electrum/Cargo.toml

```
@@ -24,3 +24,8 @@ bdk_chain = { path = "../chain" }
 default = ["use-rustls"]
 use-rustls = ["electrum-client/use-rustls"]
 use-rustls-ring = ["electrum-client/use-rustls-ring"]
+
+[[test]]
+name = "use-rustls"
+path = "tests/test_electrum.rs"
+required-features = ["use-rustls"]
```

Can we do this? - use the same name as the path
```suggestion
[[test]]
name = "test_electrum"
required-features = ["use-rustls"]
```

---

Yes, I pushed an update with it in [9b7b195](https://github.com/bitcoindevkit/bdk/pull/1615/commits/9b7b19586e1f74613f71b6c2403a7df419e13b65)